tests/CMakeLists.txt

```diff
@@ -178,11 +178,10 @@ if (NOT BYO_CRYPTO)
     # Badssl - Secure uncommon suite
     # We skip 10000san for now as its unclear the point or relevance especially with respect to the OS-based
     # TLS implementations
-    # We skip 1000san because the public badssl certificate is expired and we haven't migrated to internal hosting yet
+    # We skip 1000san, sha384 and sha512 because the public badssl certificate is expired and we haven't migrated to
+    # internal hosting yet
     # We also defer the incomplete chain test for now until we can do some further study on how to get it to
     # properly fail on windows and osx.
-    add_net_test_case(tls_client_channel_negotiation_success_sha384)
-    add_net_test_case(tls_client_channel_negotiation_success_sha512)
```

Comment:
trivial: I feel like commenting these tests is the better thing to do, since it's just that badssl.com needs to update their certificates for these websites, then we can run the tests again

(or host our own badssl)